test/e2e/OWNERS

```diff
@@ -0,0 +1,16 @@
+filters:
+  # special checks that run after the entire suite
+  # must be specially reviewed
+  "e2e_invariants.go":
+    approvers:
```

Comment:
Does this guarantee that top level approver are not able to approve?

Reply:
No, but this config guarantees auto assignment will go to this group, and there are comments throughout that you must ask.

if we want to enforce that we can refactor a bit to put the key details in a subdirectory and use no_parent_owners.

We already have no_parent_owners at the test/ level but there are a lot of people listed there.
Anyone there could already approve arbitrary changes to the test binary to add something like this, so that level of paranoia seemed unnecessary.

Reply:
Let's use a sub-directory, with or without no_parent. The way it is now unit tests for the code get linked into e2e.test. A separate package is just cleaner.

Reply:
See https://github.com/kubernetes/kubernetes/pull/133844 for my fork of this PR such that it checks for data races in log output instead of metrics.